datasource_test.go

```diff
@@ -73,3 +73,34 @@ func TestNewDataSource(t *testing.T) {
 		t.Error("datasource creation response should return the created datasource ID")
 	}
 }
+
+func TestNewPrometheusDataSource(t *testing.T) {
+	server, client := gapiTestTools(200, createdDataSourceJSON)
+	defer server.Close()
+	//client_acc, _ := New("admin:admin", "http://localhost:3000")
```

Comment:
Do the commented lines need to be in here?

Reply:
Removed